fe/fe-core/src/main/java/com/starrocks/master/ReportHandler.java

```diff
@@ -113,6 +123,10 @@ public Long getValue() {
             }
         };
         MetricRepo.addMetric(gaugeQueueSize);
+        pendingTaskMap.put(ReportType.TABLET_REPORT, Maps.newHashMap());
+        pendingTaskMap.put(ReportType.DISK_REPORT, Maps.newHashMap());
+        pendingTaskMap.put(ReportType.TASK_REPORT, Maps.newHashMap());
+        pendingTaskMap.put(ReportType.WORKGROUP_REPORT, Maps.newHashMap());
     }
 
     public TMasterResult handleReport(TReportRequest request) throws TException {
```

Comment:
Is it possible that the request contains reports of both task & disk? The original code seems to handle this situation properly, yet after this PR every ReportTask is deduplicated by its last ReportType.

Reply:
It's not possible since it has checked BE code